src/tests/ftest/container/Attribute.py

```diff
@@ -120,6 +122,7 @@ def tearDown(self):
             if self.container:
                 self.container.close()
         finally:
+            AgentUtils.stop_agent(self.hostlist, self.agent_session)
```

Comment:
Only do this if `self.agent_session is not `None`:
```suggestion
            if self.agent_session:
                AgentUtils.stop_agent(self.hostlist, self.agent_session)
```